howto-gallery/scripts/Equipment.ts

```diff
@@ -38,9 +38,7 @@ export function givePlayerEquipment(log: (message: string, status?: number) => v
   equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Mainhand, new mc.ItemStack("minecraft:wooden_sword"));
   equipmentCompPlayer.setEquipment(mc.EquipmentSlot.Offhand, new mc.ItemStack("minecraft:shield"));
 
-  const equipmentCompArmorStand = armorStand.getComponent(
-    "equippable"
-  ) as mc.EntityEquippableComponent;
+  const equipmentCompArmorStand = armorStand.getComponent("equippable") as mc.EntityEquippableComponent;
```

Comment:
This _might_ not need the `as` cast anymore?

Reply:
Correct, removed at all possible places